libs/librepcbcommon/application.cpp

```diff
@@ -33,9 +33,47 @@ namespace librepcb {
  *  Constructors / Destructor
  ****************************************************************************************/
 
-Application::Application(int& argc, char** argv) :
+Application::Application(int& argc, char** argv) noexcept :
     QApplication(argc, argv)
 {
+    // set application version
+    mAppVersion = Version(APP_VERSION);
```

Comment:
Only because this way the initialization of members is grouped together with it's following checks/asserts...